application/web/index.html

```diff
@@ -42,7 +42,18 @@
                         <label class="new_entry">Create New Entry</label>
                     </div>
                 </button>
-
+                <div class = "home-list">
+                    <div class="home-entry-descriptor">
+                        <span>Entry Name</span>
+                        <span>Date Logged</span>
```

Comment:
can you add the font-family: 'Inter' for these two texts

Reply:
it will be in the next commit, is there a visual difference by the way? I couldn't see it on the live

Reply:
yupp at least for me. It looked more like the MVP design if I add the font-family

Reply:
inside of each span, you can add this code

style="font-family: 'Inter'"

Reply:
<span style="font-family: 'Inter'">Entry Name</span>
<span style="font-family: 'Inter'">Date Logged</span>